web/controllers/password_controller.ex

```diff
@@ -0,0 +1,37 @@
+defmodule CodeCorps.PasswordController do
+  use CodeCorps.Web, :controller
+
+  alias CodeCorps.{Emails, Mailer, User, AuthToken}
+  alias Ecto.Changeset
+
+  @doc"""
+  forgot_password should take an email and generate an AuthToken model and send an email
+  """
+  def forgot_password(conn, %{"email" => email}) do
+    with %User{} = user <- Repo.get_by(User, email: email),
+        %Changeset{valid?: true} <- AuthToken.changeset(%AuthToken{}, user) do
+
+        try_create_forgot_password(user)
+        |> send_forgot_password_email
+
+        conn
+        |> put_status(200)
+        |> render("show.json", email: user.email)
+
+    else
+      nil ->
```

Comment:
@begedin pretty sure I can just get rid of this `with/else` block if always send 200.  Thoughts?